### PR TITLE
Add kOps as cilium user

### PR DESCRIPTION
**Pull request by olemarkus**
Adding kOps to users list as requested by @xmulligan 
xref https://github.com/kubernetes/kops/pull/12919#issuecomment-1043167583

### CODE REVIEW

**Comment by xmulligan** [comment]
Thanks for adding @olemarkus ! 🚀

**Review by xmulligan — APPROVED** [review]

**Review by pchaigno — APPROVED** [review]

File: USERS.md
Comment (kaworu) [inline USERS.md]: typo?
```suggestion
      D: kOps is using Cilium as one of the supported CNIs
```

**Review by kaworu — CHANGES_REQUESTED** [review]

File: USERS.md
Comment (borkmann) [reply USERS.md]: @olemarkus Do you have a chance to fix up the typo in your commit? Would be great to get it merged :) Thanks!

Comment (olemarkus) [reply USERS.md]: Certainly

Comment (borkmann) [reply USERS.md]: Awesome, thanks so much!

Comment (borkmann) [reply USERS.md]: @olemarkus One final tiny ask.. looks like there is a merge conflict, do you have a chance to rebase your change to latest master? Once that is complete I can hit merge button. :)

Comment (olemarkus) [reply USERS.md]: Sorry I missed that. Done.

**Comment by xmulligan** [comment]
Thanks @olemarkus 🐝 Let me know if you want some Cilium stickers and I can send them to you. Just DM your address on Twitter

**Pull request merged**
